pkg/proc/disasm.go

```diff
@@ -92,7 +92,7 @@ func firstPCAfterPrologueDisassembly(p Process, fn *Function, sameline bool) (ui
 			continue
 		}
 		if checkPrologue(text, prologue) {
-			r := &text[len(prologue)]
+			r := text[len(prologue)]
```

Comment:
I don't see why I should want to make a copy of the instruction.

Reply:
Thanks. After disassembly,
it is indeed found that some assembly codes will be added if symbol "&" is not used.